mayhem/extract_fuzz_string.mayhemfile

```diff
@@ -0,0 +1,4 @@
+project: Istio
```

Comment:
This isn't Istio?

Reply:
That's a nice catch. I'll fix this up. I think I was working on this in parallel with the istio repository, as they were similar.